src/discordtipbot/discordbot.cr

```diff
@@ -553,9 +553,11 @@ class DiscordBot
 
     users = active_users(msg)
 
-    amount_each = amount/people
+    return reply(msg, "**ERROR**: There aren't any people to give your money to!") unless users && (users = users.to_a).size > 0
```

Comment:
`There aren't any people to glucky on` might be better. Especially since this is crypto